### PR TITLE
Now we'll display maximum ram and vram used.

### PR DESCRIPTION
**Pull request by NinjaDanz3r**

### CODE REVIEW

**Comment by NinjaDanz3r** [comment]
iamheman

**Comment by NinjaDanz3r** [comment]
iamheman

**Comment by NinjaDanz3r** [comment]
iamheman

File: src/Game/main.cpp [hunk 1]
```diff
@@ -133,6 +133,8 @@ int main(int argc, char* argv[]) {
             myfile << "Percentage of bad frames: " << (numberOfBadFrames / static_cast<double>(numberOfFrames))*100.0 << "%\n";
             myfile << "Average frame time: " << averageFrameTime << " ms\n";
             myfile << "Max frame time: " << maxFrameTime * 1000.0 << " ms\n";
+            myfile << "Max ram used: " << maxRamUsed << "\n";
```
Comment (Chainsawkitten) [inline src/Game/main.cpp]: Output which unit? (MiB)

**Pull request merged**